mobile/cmp/dialog/Dialog.js

```diff
@@ -17,12 +17,12 @@ export const [Dialog, dialog] = hoistCmp.withFactory({
     className: 'xh-dialog',
     model: false,
 
-    render({className, isOpen, onCancel, icon, title, content, buttons = []}) {
+    render({className, isOpen, isCancelable, onCancel, icon, title, content, buttons = []}) {
```

Comment:
The prop will always receive a value from the model via the allowEscape property, so a default value should be unnecessary. The prop was previously always set to 'true', this allowed all popups to be escaped, even alerts. Since allowEscape is based on the presence of cancelProps, and alerts do not have any, I don't believe there is a way to completely preserve the previous behavior. At this point, the mobile and desktop versions behave identically.

Reply:
Let's discuss -- Dialog is actually a public component used potentially well beyond our new message model, in apps, and other hoist uses. The usage by Message is just one of many.

Reply:
I agree with @lbwexler that we should default this to true here to keep the mobile Dialog's behaviour the same. Then the only difference would be that Alerts cannot be cancelled, which I think is appropriate.

Reply:
Agreed and incorporated.